lib/exirc/client.ex

```diff
@@ -547,12 +548,13 @@ defmodule ExIrc.Client do
     {:noreply, new_state}
   end
   # Called when another user joins a channel the client is in
-  def handle_data(%IrcMessage{:nick => user_nick, :cmd => "JOIN"} = msg, state) do
+  def handle_data(%IrcMessage{:nick => user_nick, :cmd => "JOIN", :host => host, :user => user} = msg, state) do
```

Comment:
Could you also change this to the `%{key: value}` syntax? Just merged a PR which made that map style consistent everywhere and I wouldn't want to mess it up after all the work they did. Sorry about that :(